src/pages/Home.jsx

```diff
@@ -1,38 +1,264 @@
 // React Imports
-import React from 'react';
+import React from "react";
 // Material UI Imports
-import Box from '@mui/material/Box';
-import Container from '@mui/material/Container';
-import Paper from '@mui/material/Paper';
-
+import Box from "@mui/material/Box";
+import Container from "@mui/material/Container";
+import Grid from "@mui/material/Grid";
+import Button from "@mui/material/Button";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import { Typography, Stack } from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+import SecurityIcon from "@mui/icons-material/Security";
+import Diversity1Icon from "@mui/icons-material/Diversity1";
+import SupportIcon from "@mui/icons-material/Support";
 /**
  * Home - First Page you encounter in PASS before login.
  * Should not display if you are already logged in
  *
- * @memberof Pages
+ * @memarginBottomerof Pages
  * @name Home
  * @returns {React.ReactElement} The home page
  */
-const Home = () => (
-  <Container component="main" maxWidth="s">
-    <Box
-      sx={{
-        marginTop: 18,
-        marginBottom: 18,
-        display: 'flex',
-        flexDirection: 'column',
-        alignItems: 'center',
-        textAlign: 'center',
-        padding: '20px'
-      }}
-    >
-      <Paper elevation={2} sx={{ display: 'inline-block', mx: '2px', padding: '20px' }}>
+const Home = () => {
+  const theme = useTheme();
+  const isReallySmallScreen = useMediaQuery(theme.breakpoints.down("sm"));
+  return (
+    <Container component="main">
+      <Box
+        sx={{
+          margin: "18px 0",
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "center",
+          textAlign: "center",
+          padding: "20px"
+        }}
+      >
         <section id="home">
-          <h1>HOME PAGE</h1>
+          <Stack
+            width={isReallySmallScreen ? 1 : 3 / 3}
+            alignItems="center"
+            justifyContent="center"
+          >
+            <Grid item xs={12}>
+              <Box
+                component="img"
+                src="./src/assets/web_security.png"
+                alt="web-security-image"
+                sx={{
+                  width: "90%", // Default to 90%
+                  "@media (min-width:600px) and (max-width:959px)": {
+                    width: "50%"
+                  },
+                  "@media (min-width:960px) and (max-width:1023px)": {
+                    width: "25%"
+                  },
+                  "@media (min-width:1024px) and (max-width:1439px)": {
+                    width: "45%"
+                  },
+                  "@media (min-width:1440px)": {
+                    width: "45%"
+                  }
+                }}
+              />
+              <Typography
+                sx={{
+                  color: "temporaryPurple.dark",
+                  fontSize: { xs: "2rem", sm: "2rem", md: "3rem" },
```

Comment:
not sure this is optimized... I'd have to look at our other code, but not sure if you need the `sm`.   Happens a lot in this file.

Reply:
What I mean by this is that I believe MUI to be "mobile-first" like Tailwind and Bootstrap. It may stay at that size until you tell it otherwise.  But again, I'm not positive.    I may be mixing it up with MUI media queries.

Reply:
Think this was what I mentioned with regards to using both `xs` and `sm`. Un-resolving this. I don't think this has been addressed?